### ISSUE TITLE
Unwritten cloud AIP: Non-project resources

### ISSUE THREAD

**Issue by robcapo** [googleapis/api-linter#177]
We still need a cloud AIP for this one.

Old lint rule: https://g3doc.corp.google.com/google/api/tools/linter/g3doc/rules/non-project-resources.md?cl=head

**Comment by apasel422** [comment]
I'd be interested in doing this, and I have access to the original Markdown file, but what AIP does this correspond to? I assume this is blocked on a separate issue in https://github.com/aip-dev/google.aip.dev?

**Comment by noahdietz** [comment]
_Going through old issues._

That rule/guidance is no longer enforced.
